src/test/java/com/google/jenkins/plugins/credentials/domains/DescribableDomainRequirementProviderTest.java

```diff
@@ -77,9 +77,15 @@ public String getDisplayName() {
     }
   }
 
+  private AutoCloseable openMocks;
   @Before
   public void setUp() throws Exception {
-    MockitoAnnotations.initMocks(this);
```

Comment:
opens mocks and then immediately closes them.
changed to only close after the test. (the test works without it.